demo-shell/src/app/components/datatable/drag-and-drop/datatable-dnd.component.html

```diff
@@ -1,5 +1,5 @@
 <h1>DataTable Drag and Drop Demo</h1>
-<div
+<div data-automation-id="datatable"
```

Comment:
I don't see this data-automation-id used anywhere, can it be removed ?

Reply:
ok

Reply:
It is needed as we are calling the common DataTablePage. and the default locator it looks for in the page, is the above.